test/powershell/Modules/Microsoft.PowerShell.Utility/Out-File.Tests.ps1

```diff
@@ -127,4 +127,11 @@ Describe "Out-File" -Tags "CI" {
 	# reset to not read only so it can be deleted
 	Set-ItemProperty -Path $testfile -Name IsReadOnly -Value $false
     }
+    It "Creates a file using Out-File with the alias 'Path' for the -filepath parameter" {
```

Comment:
It would be good to use the header format that is used throughout the test file.